### PR TITLE
added skill request screen and the bottom navigator

### PR DESCRIPTION
**Pull request by ThomasLiARDJAVA**
# Description

Started with skill impact request by adding the stack for both conservationist and supporter. As well as added bottom tabs for navigation

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: navigation/MainNavigation.js [hunk 1]
```diff
@@ -39,12 +39,14 @@ import ResetPasswordScreen from '../screens/ResetPasswordScreen';
 import WelcomeScreen from '../screens/WelcomeScreen';
 import ConnectionsScreen from '../screens/Connections/ConnectionsScreen';
 import SelectedConnectionsScreen from '../screens/Connections/SelectedConnectionsScreen';
+import SkillRequestScreen from '../screens/SkillRequestScreen';
```
Comment (dacruz21) [inline navigation/MainNavigation.js:42]: Can we call this tab the "Skilled Impact" tab? IMO this makes it sound like it's only for applications when there's also a lot of profile config, groups, etc.

So can we basically just replace `SkillRequest` with `SkilledImpact` everywhere in this file?

File: navigation/MainNavigation.js [hunk 2]
```diff
@@ -115,6 +117,33 @@ const FeedStack = createStackNavigator(
...
 );
 
+const SkillRequestsStack = createStackNavigator(
+  //TODO add the rest of skill request screens for conservationists here
+  {
+    SkillImpactRequest: SkillRequestScreen,
+  },
+  {
+    navigationOptions: {
+      tabBarLabel: 'Skilled Impact Requests',
```
Comment (dacruz21) [inline navigation/MainNavigation.js:127]: If above sounds fine, we can drop Requests from this label and just call it "Skilled Impact"

File: navigation/MainNavigation.js (hunk 1 above)
Comment (ThomasLiARDJAVA) [reply navigation/MainNavigation.js:42]: You mean call this screen the "SkilledImpactScreen"? Just for clarification.

Comment (dacruz21) [reply navigation/MainNavigation.js:42]: Yeah I think that would be best. As well as renaming `SkillRequestsStack` to `SkilledImpactStack`, etc.

Comment (ThomasLiARDJAVA) [reply navigation/MainNavigation.js:42]: Got it. Will do.

**Pull request merged**